nemo_text_processing/text_normalization/es/taggers/electronic.py

```diff
@@ -36,39 +52,55 @@ def __init__(self, deterministic: bool = True):
         super().__init__(name="electronic", kind="classify", deterministic=deterministic)
 
         dot = pynini.accep(".")
-        accepted_common_domains = pynini.union(*common_domains)
-        accepted_symbols = pynini.union(*symbols) - dot
-        accepted_characters = pynini.closure(NEMO_ALPHA | NEMO_DIGIT | accepted_symbols)
-        acceepted_characters_with_dot = pynini.closure(NEMO_ALPHA | NEMO_DIGIT | accepted_symbols | dot)
+
+        symbols = [x[0] for x in load_labels(get_abs_path("data/electronic/symbols.tsv"))]
+        symbols = pynini.union(*symbols)
+        symbols_no_period = pynini.difference(symbols, dot)
```

Comment:
Use `full_stop` not `dot` Less ambiguity

Reply:
Same for instances of period

Reply:
Renamed `dot` to `full_stop` as requested.

`period` is a string literal for `.`